users/luescher/setups/librispeech/pipeline_base_args.py

```diff
@@ -675,10 +690,18 @@ def get_data_inputs(
         "normalize_pronunciation": False,
     }
 
+    g2p_train_apply_args = {}
```

Comment:
I do not understand why passing the g2p args is needed, because either:
 -  someone is running the pipeline at i6, in this case all g2p jobs already exist and can be imported
 - someone outside is running the pipeline, but then it is no issue to have g2p installed in the Sisyphus env, just as the `requirements.txt` specifies.

Reply:
at the moment you cannot pass arguemtns to the g2p at all.. which probably is also not necessary.. so maybe this is a bit over-kill ^^
also remove `g2p_&_args` from `get_g2p_augmented_bliss_lexicon_dict`?

Reply:
I would suggest yes, we should avoid that people use different G2P.